ovos_bus_client/client/async_client.py

```
@@ -0,0 +1,400 @@
+                if not retry:
+                    raise
+                await asyncio.sleep(retry_delay)
+
+    async def _recv_loop(self):
+        """Receive loop — runs until the connection closes."""
+        try:
+            async for raw in self._ws:
+                await self._on_message(raw)
+        except (ConnectionClosedOK, ConnectionClosedError):
+            pass
+        except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)